src/main/java/wromaciej/hvac_sim/thermo/Ids.java

```diff
@@ -0,0 +1,28 @@
+package wromaciej.hvac_sim.thermo;
+
+import wromaciej.hvac_sim.display.Item;
+
+import java.util.HashMap;
+import java.util.Set;
+
+public class Ids {
+
+    private static HashMap<Integer, Item> items;
```

Comment:
I imagine you are going to use some kind of flyweight pattern here. Be careful, if you will store many objects in memory you may end up in crashing app because of memory leak.